display_list/dl_color.h

```diff
@@ -58,58 +77,99 @@ struct DlColor {
   static constexpr DlColor kCornflowerBlue()     {return DlColor(0xFF6495ED);};
   // clang-format on
 
-  constexpr bool isOpaque() const { return getAlpha() == 0xFF; }
-  constexpr bool isTransparent() const { return getAlpha() == 0; }
+  constexpr bool isOpaque() const { return alpha_ >= 1.f; }
+  constexpr bool isTransparent() const { return alpha_ <= 0.f; }
+
+  constexpr int getAlpha() const { return toC(alpha_); }
```

Comment:
Some docs on what these do about floats not in the [0,1] range maybe?

Reply:
Done, added deprecation notice.